README.md

```diff
@@ -1,2 +1,76 @@
-# my_hello_microverse
-Hello microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+Hello Microverse
+
+> Description the project.
+Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
+Added descriptive README file my project.
+Add 3 files each one in a separate commit:
+A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
+A .css file that includes one class with styles for your header (make it in your favorite color).
+A test.md file with the text "This file should be ignored by git".
+
+## Built With
+HTML AND CSS
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@emekaemmanuel](https://github.com/emekaemmanuel)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/emeka-ugboaja-167820226)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/EmekaEmmanuel/hello_microverse/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
Kindly use the new and [updated README file](https://github.com/microverseinc/readme-template) provided by Microverse. ALl main headers from this README template should be present in a student's repo. The only optional section is FAQ.